### ISSUE TITLE
Arbitrary warning for png images of format RGB w/8bit channels without alpha

### ISSUE THREAD

**Issue by heydojo** [Vhati/Slipstream-Mod-Manager#12]
Hi,
When validating a mod containg png images, a warning will appear for png images which do not have an alpha channel:
`Type: RGB w/8bit channels (Usually RGB+Alpha w/8bit channels)`

Could this warning please be suppressed for RGB w/8bit png images?
png images do not require an alpha channel to work in FTL. There is actually no point in adding an alpha channel to an image which will be used as a background. Adding alpha to background images needlessly increases the file size.
png images without an alpha channel work just fine in FTL.
Thanks.

**Comment by Vhati** [comment]
> png images do not require an alpha channel to work in FTL.

This is true, hence 'usually' and merely a nag warning. That was just reporting a norm.
Even the vanilla main menu background ( "/img/main_menus/main_base2.png" ) has an alpha.

Then again, I doubt anyone who actually needs transparency - yet forgot it - would get so far into development as to benefit from that reminder to have an alpha channel. So it IS rather spammy.

**Comment by heydojo** [comment]
+1
